doc/metrics.md

```diff
@@ -0,0 +1,113 @@
+## Xline metrics
+
+Many metrics are similar to those in [etcd](https://etcd.io/docs/v3.5/metrics/).
+
+### CURP Server
+
+1. `leader_changes`: Counter
+The number of leader changes seen.
+
+2. `learner_promote_failed`: Counter
+The total number of failed learner promotions (likely learner not ready) while this member is leader.
+
+3. `learner_promote_succeed`: Counter
+The total number of successful learner promotions while this member is leader.
+
+4. `heartbeat_send_failures`: Counter
+The total number of leader heartbeat send failures (likely overloaded from slow disk).
+
+5. `apply_snapshot_in_progress`: UpDownCounter
+not equals to 0 if the server is applying the incoming snapshot. 0 if none.
+
+1. `proposals_committed`: ObservableGauge
```

Comment:
why are the bullet numbers below all 1s?

Reply:
Sorry, I didn't notice that Markdown rendered it correctly. It is fixed.